### ISSUE TITLE
Bug report: uncaught exception in EQTL test case

### ISSUE THREAD

**Issue by dongbohu** [greenelab/cimr#60]
The bug is found on both `master` and `polish_v_0_1_6` branches.
--
When running this EQTL test case:
https://github.com/greenelab/cimr/tree/polish_v_0_1_6/examples/eqtl
with chunksize 5000, the following line
 https://github.com/greenelab/cimr/blob/607730b914072bf07f3a347c1fa21b8b2eaae2b3/cimr/processor/process.py#L314
raises an exception, because `self.data['feature_id'][0]` doesn't exist. After checking the intermediate results, I found that the first row in `self.data` DataFrame (based on chunk #7) has been removed, so the value in the first row's `feature_id` column of `self.data` is actually `self.data['feature_id'][1]`. 

The bug can be fixed by:
```
if self.data['feature_id'].iloc[0].startswith('ENS'):
    ...
```

**Comment by dongbohu** [comment]
Followup:
--
* The decompressed input file `submitted_data/eqtl/eqtl.txt` has 40,119 rows (including the header line). 
* When chunksize is 5 million (default), decompressed output file has 34,731 lines (including header);
* When chunksize is 10,000, decompressed output file has 34,720 lines (including header);

**Comment by dongbohu** [comment]
Fixed by PR #64.
